pkg/generated/openapi/BUILD

```diff
@@ -8,6 +8,8 @@ openapi_library(
     name = "go_default_library",
     srcs = ["doc.go"],
     openapi_targets = [
+        "_output/dockerized/go/src/github.com/miekg/coredns/vendor/k8s.io/client-go/1.5/pkg/runtime",
```

Comment:
I noticed these two lines were added by code generation and I removed them manually in #49983. Should they be included?